programs/server/config.xml

```diff
@@ -742,6 +742,10 @@
              It also enables 'changeable_in_readonly' constraint type -->
         <settings_constraints_replace_previous>true</settings_constraints_replace_previous>
 
+        <!-- By default, for backward compatibility create table with a specific table engine ignores grant,
+             however you can change this behaviour by setting this to true -->
+        <table_engines_require_grant>false</table_engines_require_grant>
```

Comment:
Usually we activate access control improvements for our stateless tests - see the [tests/config/config.d/enable_access_control_improvements.xml](https://github.com/ClickHouse/ClickHouse/blob/master/tests/config/config.d/enable_access_control_improvements.xml) file. Of course doing so might also require to modify some stateless tests.